app/pulp/app/models/auth.py

```diff
@@ -0,0 +1,63 @@
+"""
+Django models related to the Authentication system
+"""
+from django.core import validators
+from django.contrib.auth.models import BaseUserManager, AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+
+
+class PulpUser(AbstractBaseUser, PermissionsMixin):
```

Comment:
Can this be `User`? It usage in `settings.py` demonstrates the redundancy of including 'Pulp' in its name.